examples/smac/feedforward/decentralised/run_ippo_smac_death_masking.py

```diff
@@ -92,10 +94,21 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         policy_optimiser=policy_optimiser,
         critic_optimiser=critic_optimiser,
         run_evaluator=True,
-        epoch_batch_size=5,
+        epoch_batch_size=32,
         num_epochs=15,
         num_executors=1,
-        multi_process=True,
+        multi_process=False,
+        sequence_length=121,
+        period=121,
```

Comment:
Did we decide that this is the only fix to our problem? There was no solution that still allows for smaller sequences. I think the code uses smaller sequences, but the full sequence is used to calculate GAE? Or do they also train on the full sequence?

Reply:
We probably need to run an experiment to check that but in that case we will be computing truncated GAE that do not benefit from the full episode advantages.  In the onpolicy implementation they do the truncation in shorter sequences for the recurrent network after computing the GAE and the returns.

Reply:
Agreed. Maybe in the future we can figure out a way to still use shorter sequences without affecting the GAE.